runtime/flang/iso_fortran_env.f90

```diff
@@ -59,14 +59,14 @@ module ISO_FORTRAN_ENV
 	integer REAL64
 	parameter (REAL64 = 8)
 	integer REAL128
-	parameter (REAL128 = -1)
+	parameter (REAL128 = 16)
```

Comment:
shouldn't this change be guarded by #ifdef's?

Reply:
Thanks for the catch, Done.